cli/src/main/resources/log4j2.xml

```diff
@@ -9,6 +9,12 @@
     <Root level="warn">
       <AppenderRef ref="Console"/>
     </Root>
+    <Logger name="org.apache.http.headers" level="off">
```

Comment:
What's the reason behind turning these loggers completely off instead of setting the level to `WARN` or `ERROR`?

Reply:
I was probably just so annoyed by these loggers that I wanted to kill them off completely 😉 But you're right, setting the level to "error" should be enough. will do.